CONTRIBUTING.md

```diff
@@ -145,31 +145,50 @@ Branch from master:
 git checkout -b myfeature
 ```
 
-Then edit code on the `myfeature` branch.
+### Step 4: Develop
 
-#### Build
+#### Edit the code
+
+You can now edit the code on the `myfeature` branch.
+
+#### Run stand-alone mode
+
+If you want to reproduce and investigate an issue, you may need
+to run TiDB in stand-alone mode.
+
+```sh
+# Build the binary.
+make server
+
+# Run in stand-alone mode. The data is stored in `/tmp/tidb`.
+bin/tidb-server
+```
+
+Then you can connect to TiDB with mysql client in another session.
```

Comment:
What mean by `in another session`?

Reply:
another terminal

Reply:
"Then you can connect to TiDB with mysql client." is enough.